guides/source/security.md

```diff
@@ -1206,6 +1206,20 @@ TIP: Learn more about credentials with `bin/rails credentials:help`.
 
 WARNING: Keep your master key safe. Do not commit your master key.
 
+#### Accessing environment specific credentials file
+
+In order to access environment specific credential file, run `bin/rails credentials:edit --environement=environment_name`
+
+For example, to edit a production credential file, you need to run
+```sh
+bin/rails credentials:edit --environment=production
+```
+
+In a non-development environment, a warning message will be shown if you are editing/accessing the credential file of an environment that isn't your `RAILS_ENV`. i.e, when you try to access production credential file from your staging environment setup, you will see the following warning message. 
+
+> You are accessing production credentials from the staging environment. Did you mean to run `rails credentials:edit --environment staging`? For more information try `rails credentials:help`
```

Comment:
I don't think this part is necessary. Let's show the user the correct way to do it, they will see the warning if they get it wrong.

Reply:
Uhm, @ghiculescu mind sharing a little more context please? So which part do you think is not necessary. I mean, I am currently showing the warning message when the `rails credentials:edit` or `rails credentials:show` is ran under certain scenario. I am little bit confused about the part that is not necessary. 😅

Reply:
(I'm not a core committer so ultimately it's not up to me, but) I don't think lines 1218 - 1220 are necessary. I think it's better to keep the docs concise and tell you what you need to know to do the task correctly.

Reply:
Oh, got it. I'll reword the added docs to be more concise. Thanks!! 😄

Reply:
@ghiculescu Made the docs more concise. Now it delivers only the needed information.